util/dvsim/SimCfg.py

```diff
@@ -672,8 +673,8 @@ def create_bucket_report(buckets):
 
         self.results_md = results_str
 
-        # Write results to the scratch area
-        results_path = self.scratch_path + "/results_" + self.timestamp + ".md"
+        # Write results to the scratch report area
```

Comment:
I dont think anyone has ever looked at this md file. We should simply create the HTML file instead (which we will publish to the report server).

Reply:
sure, I am fine with deleting this `.md` file.